src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -799,8 +800,8 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
     return (
       <Button
         key={buttonConfig.type}
-        onClick={clickFunc}
-        label={buttonTitle}
+        onClick={() => { if (!this.isLoginForButtonFeatureEnabled(buttonConfig)) { clickFunc(); } }}
+        label={(this.props.user.username) ? buttonTitle : `Login to ${buttonTitle}`}
```

Comment:
this will change all button text to `Login to ...`, not only for download buttons

and for export buttons, it will not redirect, which will be very confusing

Reply:
Updated this logic to redirect only for download buttons